pkg/reconciler/workload/resource/resource_reconcile_test.go

```diff
@@ -188,6 +188,9 @@ func TestComputePlacement(t *testing.T) {
 			}, map[string]string{
 				"state.workload.kcp.dev/cluster-3": "Sync",
 			}, nil, nil, "ns"),
+			wantAnnotationPatch: map[string]interface{}{
```

Comment:
@sttts do you agree that the test was previously wrong, or was it on purpose that the deletion annotation would be kept even when rescheduling to the SyncTarget ?

Reply:
not sure. Leaving that to @qiujian16 and @jmprusi